app/views/settings/index.html.erb

```diff
@@ -18,6 +18,8 @@
           <div class="actions">
             <%= f.submit "Update", class: "btn-lg btn-primary btn-block" %>
           </div>
+          <h4 class="text-center">or</h4>
```

Comment:
We need a bit more than this, this actually creates an entirely different user when you click this button!